tools/Microsoft.Build.Locator/Microsoft.Build.Locator.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{158E7760-6F87-446C-B782-18E23BF54DA6}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.Build.Locator</RootNamespace>
+    <AssemblyName>Microsoft.Build.Locator</AssemblyName>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
```

Comment:
Should the output path in fact be `tools/Microsoft.Build.Locator/bin/Debug/Microsoft.Build.Locator.dll`? More "conventional" in this project would be to to place it into its anticipated output location:

```xml
<OutputPath>..\..\bin\Debug\bin</OutputPath>
```

as it'll mostly be used by `xabuild.exe` (built into `$(topdir)/bin/$(Configuration)/bin`) and the unit tests (which will need to copy elsewhere), so setting `$(OutputPath)` to `$(topdir)/bin/$(Configuration)/bin` makes sense.